packages/inference/src/HfInference.ts

```diff
@@ -21,22 +21,83 @@ type TaskWithNoAccessTokenNoEndpointUrl = {
 	) => ReturnType<Task[key]>;
 };
 
-export class HfInference {
-	private readonly accessToken: string;
-	private readonly defaultOptions: Options;
+export class HfInference implements Task {
```

Comment:
```typescript
export class HfInference implements Task {
```

That bit makes sure `HfInference` defines a method for every task (`textToImage`, `chatCompletion`, etc)